catalog/async/catalog-async-postingestplugin/src/test/groovy/org/codice/ddf/catalog/async/processingplugin/ProcessingPostIngestPluginTest.groovy

```diff
@@ -107,6 +108,93 @@ class ProcessingPostIngestPluginTest extends Specification {
         properties << [createFalseProcessingCompleteProperties(), [:]]
     }
 
+    def 'test process CreateResponse with processResource populated from metacard'() {
+        String sourceId = "source"
```

Comment:
Might want to put all of this setup stuff in a `given:` block to be consistent with other groovy unit tests.